resources/js/langs/de/profile_dashboard/middle.json

```diff
@@ -0,0 +1,15 @@
+{
+    "radar-total-categories": "Total Categories",
+    "radar-cat-smoking": "smoking",
+    "radar-cat-food": "food",
+    "radar-cat-coffee": "coffee",
+    "radar-cat-softdrinks": "soft drinks",
+    "radar-cat-alcohol": "alcohol",
+    "radar-cat-other": "other",
+    "radar-cat-coastal": "coastal",
+    "radar-cat-sanitary": "sanitary",
+    "radar-cat-dumping": "dumping",
+    "radar-cat-industrial": "industrial",
+    "radar-cat-brands": "brands",
```

Comment:
We have already translated the categories? We do not need translation values for the key names